### ISSUE TITLE
project 4 ideas

### ISSUE THREAD

**Issue by presto2116** [ga-dc/wdidc5_project4#26]
These are just a few of them that I have brainstormed. Might have more by tomorrow. and will then hopefully be able to narrow it down from there.

Pets
A user can submit their best pet photo. And other users can flip through all the pets and rate them to see which pet is the cutest. There will be a leaderboard with weekly, monthly and alltime best. 
Admins will have the power to remove any improper photos or duplicates.

TeamTracker
Sports app that easily keeps you up to date with your favorite teams scores, schedule, news (sends alerts during close games)

Get Outside and Play
create outdoor activities such as soccer games, ultimate frisbee, and other sports. select a date and open it up to registration. If you get enought people to commit, then the activity is held. Easy way to throw out ideas and see if others are interested. Example: its Monday morning and it is pouring rain. All I want to do is play some ultimate frisbee outside and I see the weather is great on saturday. I will create an activity. Activity: Ultimate frisbee, location : Silver Spring park. Date Saturday May 9. Time: 1 to 2 pm. Then open it to the community. If enough are interested, then the creator can send email to interested parties and the event is held. If no interest, try again next week...

Get Outside and Play (idea #2)
Users can post bike routes, hiking trails, sports fields. Almost anything that has to do with outside onto a map so that other users can search in their area for something fun to do. Users then can search for their area and can also filter by sport/activity to find new ways to get outside and play.

Beer
Ever run into the problem that you are always drinking the same beer. This site will let you enter your favorite beers and then give you suggestions for what other beers are similar yet different. Also would like to be able to have the user have a page where they can enter all the beers they have tried with a comments section so they can refer back to it as they dive into the world of great beers. they can also see what others have said about the beers and rank them

**Comment by adambray** [comment]
Hey Preston,

I feel like from a learning perspective, any of these ideas are good for a project 4. That said, the Pets and Beer ideas are ones that have been done to some degree before on the web, so I would ask yourself how you might differentiate. The Beer one is interesting in that you'd get to explore similarity matching algorithms (like Amazon etc use).

The TeamTracker could be tough because sports APIs are notoriously finicky or require payment or approval. 

The get outside and play ideas I really like! I think either could demonstrate well what you've learned in the class, and feel very original. 

In short, all of them are good ideas, just pick what you want to do most, and flesh it out a bit more this week! Let me know if you have any questions.

-Adam
